auth_connection_failed = (
    "Error internal communication with provider for auth url failed."
)
